lib/core/network/api/user_account_service.dart

```diff
@@ -1,26 +1,49 @@
 import 'package:dio/dio.dart';
-import 'package:hypha_wallet/core/logging/log_helper.dart';
+import 'package:get_it/get_it.dart';
 import 'package:hypha_wallet/core/network/api/endpoints.dart';
+import 'package:hypha_wallet/core/network/api/remote_config_serivice.dart';
 import 'package:hypha_wallet/core/network/networking_manager.dart';
-import 'package:image_picker/image_picker.dart';
 
-// TODO(Nik): these API calls need your help
 class UserAccountService {
   final NetworkingManager networkingManager;
 
   UserAccountService({required this.networkingManager});
 
-  Future<Response> createUserAccount({
+  Future<bool> createUserAccount({
+    required String code,
     required String accountName,
-    required String userName,
-    XFile? image,
+    required String publicKey,
+    required String network,
   }) async {
-    // TODO(Nik): these API calls need your help
-    return await Future.delayed(const Duration(seconds: 3))
-        .then((value) => Response(data: true, requestOptions: RequestOptions(path: 'MOCK')));
-    // return await dioClient.post(Endpoints.userAccountAvailable, data: {
-    //   'userAccount': userAccount,
-    // });
+    final requestBody = '''
+    { 
+      "code": "$code",
+      "accountName": "$accountName",
+      "publicKey": "$publicKey",
+      "network": "$network"
+    }''';
+    final url = GetIt.I<RemoteConfigService>().accountCreatorEndpoint + Endpoints.creteAccount;
+    try {
+      // ignore: unused_local_variable
+      final res = await networkingManager.post(url, data: requestBody);
+
+      // TODO(gguij): Not sure we need to handle the response - how to we parse status codes?
```

Comment:
we need to parse the response and save the data into our shared pref 

Or not, and jsut log the user in and let the other pages fetch the data. 

But since we have the data we he should be nice and store it

Reply:
Yeah I will get the ppp service working and do this then